internal/guest/runtime/hcsv2/uvm.go

```diff
@@ -289,16 +294,20 @@ func (h *Host) CreateContainer(ctx context.Context, id string, settings *prot.VM
 		return nil, errors.Wrapf(err, "container creation denied due to policy")
 	}
 
-	// Export security policy as one of the process's environment variables so that application and sidecar
-	// containers can have access to it. The security policy is required by containers which need to extract
-	// init-time claims found in the security policy.
+	// Export security policy and signed UVM reference info as one of the
+	// process's environment variables so that application and sidecar
+	// containers can have access to it. The security policy is required
+	// by containers which need to extract init-time claims found in the
+	// security policy.
 	//
-	// We append the variable after the security policy enforcing logic completes to bypass it; the
-	// security policy variable cannot be included in the security policy as its value is not available
-	// security policy construction time.
+	// We append the variable after the security policy enforcing logic
+	// completes to bypass it; the security policy variable cannot be included
+	// in the security policy as its value is not available security policy
+	// construction time.
 	if oci.ParseAnnotationsBool(ctx, settings.OCISpecification.Annotations, annotations.SecurityPolicyEnv, false) {
 		secPolicyEnv := fmt.Sprintf("SECURITY_POLICY=%s", h.securityPolicyEnforcer.EncodedSecurityPolicy())
-		settings.OCISpecification.Process.Env = append(settings.OCISpecification.Process.Env, secPolicyEnv)
+		uvmReferenceInfo := fmt.Sprintf("SIGNED_UVM_REFERENCE_INFO=%s", h.signedUVMMeasurementInfo)
```

Comment:
I am worried about conflicting with existing env variables
can we prefix "HCSSHIM_" or something to these? or is it too late for that?

Reply:
sidecars might already depend on `SECURITY_POLICY` env, but we can prefix the new one. The overrides are fine here, since those two should be properly documented by package consumers/service providers.

Reply:
I was going to say, if you can't change `SECURITY_POLICY`, may as well the signed info the same (or have it be `SECURITY_POLICY_UVM_REFERENCE_INFO`).
But we can keep it as is, no need to change it again